tests/unit/utils/sort-test.ts

```diff
@@ -0,0 +1,10 @@
+import { sortCustomFormFields } from 'open-event-frontend/utils/sort';
+import { module, test } from 'qunit';
+
+module('Unit | Sort | Custom Form Fields', function() {
+  test('test custom form fields sort', function(assert) {
+    const items = [{fieldIdentifier: 'age', isComplex: false}, {fieldIdentifier: 'job', isComplex: false}, {fieldIdentifier: 'majama', isComplex: true}, {fieldIdentifier: 'name', isComplex: false}, {fieldIdentifier: 'hay', isComplex: true}, {fieldIdentifier: 'trust', isComplex: false}, {fieldIdentifier: 'company', isComplex: false}];
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5875345)